source/Mission.cpp

```diff
@@ -154,6 +155,25 @@ void Mission::Load(const DataNode &node)
 			if(child.Size() >= 3)
 				deadlineMultiplier += child.Value(2);
 		}
+		else if(child.Token(0) == "route options")
+		{
+			if(child.HasChildren())
+			{
+				for(const DataNode &grand : child)
+				{
+					if(grand.Token(0) == "no wormholes")
+						routeOptions.wormholeStrategy = WormholeStrategy::NONE;
+					else if(grand.Token(0) == "all wormholes")
```

Comment:
No option for unrestricted wormholes?

Reply:
I think that's currently the default, though maybe that should change?

Reply:
I think it's fine. I was thinking about overwriting missions later but that isn't possible anyways so ignore.

Reply:
Providing a way to explicitly set the current default value is reasonable.

Reply:
Should it also support explicitly saying `no jump drive`?

Reply:
I'd say no cause it's a boolean choice.